oaknational/Oak-Web-Application
# Feat/lesq 1011/auth terms and conditions [LESQ-1011]

## Description

Music year: 1989

- adds terms and conditions on the sign in/up pages
- removes terms and conditions from onboarding page
- fixes a bug where manual school input name wasn't being sent to hubspot

## How to test

1. Go to https://deploy-preview-2716--oak-web-application.netlify.thenational.academy/sign-up 
2. Click on `Already have an account? Sign in`
3. You should see the terms and conditions links below the sign in/up component
4. Go to  https://deploy-preview-2716--oak-web-application.netlify.thenational.academy/onboarding
5. You should see no terms and conditions on these pages

## Screenshots

How it should now look:
<img width="600" alt="Screenshot 2024-08-20 at 13 59 28" src="https://github.com/user-attachments/assets/2c4500e2-2822-46d5-ab89-8d4a8eaaecbe">
<img width="600" alt="Screenshot 2024-08-20 at 13 59 45" src="https://github.com/user-attachments/assets/b31bc115-bda9-418a-99e1-d0e805e0c2c2">


## Checklist

- [ ] Ts and cs are on the sign up and sign in pages
- [ ] The ts and cs match the designs
- [ ] the links go to the correct pages
- [ ] the links open in a new tab
- [ ] the aria label indicates the links open in a new tab
- [ ] the onboarding pages do not contain ts and cs
- [ ] when submitting a manual school input, the school is captured in hubspot
